x/wasm/keeper/addresses_test.go

```diff
@@ -3,21 +3,75 @@ package keeper
 import (
 	"encoding/json"
 	"fmt"
+	tmbytes "github.com/cometbft/cometbft/libs/bytes"
 	"testing"
 
-	tmbytes "github.com/cometbft/cometbft/libs/bytes"
 	"github.com/stretchr/testify/require"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-func TestBuildContractAddress(t *testing.T) {
+func PrepareCleanup(t *testing.T) {
```

Comment:
this function shuld be private
```suggestion
func prepareCleanup(t *testing.T) {
```

Reply:
Done.